pkg/test/ginkgo/options_monitor_events.go

```diff
@@ -72,6 +71,7 @@ func NewMonitorEventsOptions(out io.Writer, errOut io.Writer) *MonitorEventsOpti
 			RunDataWriterFunc(monitor.WriteTrackedResourcesForJobRun),
 			RunDataWriterFunc(monitor.WriteBackendDisruptionForJobRun),
 			RunDataWriterFunc(allowedalerts.WriteAlertDataForJobRun),
+			RunDataWriterFunc(monitor.WriteClusterData),
```

Comment:
I'm a little worried about this part.  I think you could still end up in the situation where cluster-data isn't empty but is incomplete, I don't know if we want to write that out to the gcs bucket if it's what sippy will consume

Sippy updates a job's variants each time it encounters a run, so you could have some churn.  Imagine for some reason topology ends up blank in one run, and populated in the next.  You'd be adding/removing the topology variant (e.g. ha, single-node) every time that happens